widget/ui/src/components/Modal/Modal.styles.ts

```diff
@@ -48,7 +48,7 @@ export const ModalContainer = styled('div', {
   display: 'flex',
   flexDirection: 'column',
   zIndex: 9999999,
-  transform: 'translateY(100%)',
+  transform: 'translateY(100vh)',
```

Comment:
It partially solves the problem. I believe the other issue with the modal component is its implementation. Using timeouts can make it buggy. The modal animation duration is 450 ms, and there's a variable called CLOSED_DELAY that refers to the time the modal will be removed from the DOM after the user action, which has a different value (600 ms). I think we can remove these timeouts and use the native `onAnimationEnd` event on the modal container. Something like this:

```ts
const Modal = ({ open } : {open: boolean}) => {
  const [isMounted, setIsMounted] = useState(open);
  const [animationClass, setAnimationClass] = useState('');

  useEffect(() => {
    if (open) {
      setIsMounted(true);
      setAnimationClass('mount');
    } else {
      setAnimationClass('unmount');
    }
  }, [open]);

  const handleAnimationEnd = () => {
    if (!open) {
      setIsMounted(false);
    }
  };

  return (
    isMounted && (
      <ModalContainer
       //Modify this to be more suitable for Stitches.js
        className={animationClass}
        onAnimationEnd={handleAnimationEnd}
      >
     .............................
      </ModalContainer>
    )
  );
};
```